### PR TITLE
security center -> investigate feature

### PR DESCRIPTION
**Pull request by jason-cf**
closes pcx-9551

### CODE REVIEW

**Review by kodster28 — APPROVED** [review]

File: content/radar/investigate/url-scanner.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-title: URL Scanner (beta)
+title: URL Scanner
 weight: 7
 ---
 
-{{<heading-pill style="beta">}} Radar's URL Scanner {{</heading-pill>}}
+# URL Scanner
 
-In order to better understand Internet usage around the world, Radar provides a URL Scanner at [https://radar.cloudflare.com/scan](https://radar.cloudflare.com/scan).
+In order to better understand Internet usage around the world, you can use Cloudflare's URL Scanner. With this too, you have the ability to investigate deeper into the details of a domain, IP, URL, or ASN. It is available in the Cloudflare Dashboard in Security Center, [Cloudflare Radar](https://radar.cloudflare.com/scan) and our [API](/api/operations/urlscanner-search-scans).
```
Comment (deadlypants1973) [inline content/radar/investigate/url-scanner.md]:
```suggestion
To better understand Internet usage around the world, use Cloudflare's URL Scanner. With Cloudflare's URL Scanner, you have the ability to investigate the details of a domain, IP, URL, or ASN. Cloudflare's URL Scanner is available in the Security Center of the Cloudflare dashboard, [Cloudflare Radar](https://radar.cloudflare.com/scan) and the Cloudflare [API](/api/operations/urlscanner-search-scans).
```

File: content/radar/investigate/url-scanner.md [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+
+## Using the API
```
Comment (deadlypants1973) [inline content/radar/investigate/url-scanner.md]:
```suggestion
## Use the API
```
no gerunds in titles

File: content/security-center/investigate/investigate-threats.md [hunk 3]
```diff
@@ -6,77 +6,56 @@ weight: 3
...
 
-    You can view and [request categorization](/security-center/investigate/change-categorization/) for a domain. Uncategorized domains can also request to have a category added. This request goes through an approval process through the Cloudflare team.
+## IP Address
 
-+ IP resolution (current)
-+ API curl
+An [IP address](https://www.cloudflare.com/learning/dns/glossary/what-is-my-ip-address/) is a unique address that identifies a server. It stands for [Internet Protocol](https://www.cloudflare.com/learning/network-layer/internet-protocol/), which is the set of rules that allows servers to communicate with each other. 
 
-### WHOIS
+IP address search allows you to search both [IPv4 and IPv6](https://www.cloudflare.com/learning/dns/glossary/what-is-my-ip-address/) addresses and retrieve relevant information such as their pointer records, AS numbers and passive DNS records.
```
Comment (deadlypants1973) [inline content/security-center/investigate/investigate-threats.md]:
```suggestion
IP address search allows you to search both [IPv4 and IPv6](https://www.cloudflare.com/learning/dns/glossary/what-is-my-ip-address/) addresses and retrieve relevant information such as their pointer records, AS numbers and passive DNS records.
```

File: content/security-center/investigate/investigate-threats.md [hunk 4]
```diff
@@ -6,77 +6,56 @@ weight: 3
...
-+ Creation date of the domain
-+ Most recent update
-+ Registrant (if available)
-+ Registrar
-+ Nameservers (if available)
-+ API curl
+## Domain
 
-### Domain history
+A [domain name](https://www.cloudflare.com/learning/dns/glossary/what-is-a-domain-name/) is a string of text that maps to an IP address. They are used to help us remember where websites are hosted. Domain names are purchased through [Registrars](/registrar/) and can be acquired easily by anyone.
```
Comment (deadlypants1973) [inline content/security-center/investigate/investigate-threats.md]:
```suggestion
A [domain name](https://www.cloudflare.com/learning/dns/glossary/what-is-a-domain-name/) is a string of text that maps to an IP address. Domain names are used to help people remember where websites are hosted. Domain names are purchased through [registrars](/registrar/) and can be acquired easily by anyone. 
```
Registrar is a product but the concept being refered to here is the general registrar hence lowercase

File: content/security-center/investigate/investigate-threats.md [hunk 5]
```diff
@@ -6,77 +6,56 @@ weight: 3
...
-+ API curl
+## Domain
 
-### Domain history
+A [domain name](https://www.cloudflare.com/learning/dns/glossary/what-is-a-domain-name/) is a string of text that maps to an IP address. They are used to help us remember where websites are hosted. Domain names are purchased through [Registrars](/registrar/) and can be acquired easily by anyone. 
 
-+ Category
-+ Changed On
-+ API curl
+By searching for a domain name, Cloudflare will provide an overview with the domain's category and IP addresses it currently resolves to.
```
Comment (deadlypants1973) [inline content/security-center/investigate/investigate-threats.md]: Who is searching for the domain name? It sounds like Cloudflare is searching and I don't think you meant that here.

File: content/security-center/investigate/investigate-threats.md [hunk 6]
```diff
@@ -6,77 +6,56 @@ weight: 3
...
 
-### Overview
+A domain can have multiple categories. Cloudflare displays both the parent category and the detailed child category. You can [request category changes](/security-center/investigate/change-categorization/) for a domain. Miscategorized domains can also request to have a category added. This request goes through an approval process with the Cloudflare team.
 
-+ Type (IPv4 / IPv6)
-+ ASN Info
+{{</Aside>}}
 
-### Passive DNS Records
+As part of the domain search results, we show the WHOIS details and a history of its category changes over time.
```
Comment (deadlypants1973) [inline content/security-center/investigate/investigate-threats.md]:
```suggestion
As part of the domain search results, Cloudflare show the WHOIS details and a history of its category changes over time. 
```

File: content/security-center/investigate/investigate-threats.md [hunk 7]
```diff
@@ -6,77 +6,56 @@ weight: 3
...
-### Passive DNS Records
+As part of the domain search results, we show the WHOIS details and a history of its category changes over time. 
 
-+ Hostname
-+ First seen timestamp
-+ Last seen timestamp
+## AS Number
 
-## AS Number search
+An [AS number](https://www.cloudflare.com/learning/network-layer/what-is-an-autonomous-system/) is a group of IP addresses belonging and controlled by a single organization. The entire group of networks have a single unified routing policy. The [Internet Assigned Numbers Authority](https://www.iana.org/) (IANA) is the organization responsible for managing the assignment and distribution of AS numbers. The AS number's routing policies are used by [BGP](https://www.cloudflare.com/learning/security/glossary/what-is-bgp/) which is how Cloudflare's [anycast network](https://www.cloudflare.com/learning/cdn/glossary/anycast-network/) works.
```
Comment (deadlypants1973) [inline content/security-center/investigate/investigate-threats.md]:
```suggestion
An [AS number](https://www.cloudflare.com/learning/network-layer/what-is-an-autonomous-system/) is a group of IP addresses belonging to and controlled by a single organization. The entire group of networks have a single unified routing policy. The [Internet Assigned Numbers Authority](https://www.iana.org/) (IANA) is the organization responsible for managing the assignment and distribution of AS numbers. The AS number's routing policies are used by [BGP](https://www.cloudflare.com/learning/security/glossary/what-is-bgp/) which is how Cloudflare's [anycast network](https://www.cloudflare.com/learning/cdn/glossary/anycast-network/) works. 
```

File: content/security-center/investigate/investigate-threats.md [hunk 8]
```diff
@@ -6,77 +6,56 @@ weight: 3
...
-+ Hostname
-+ First seen timestamp
-+ Last seen timestamp
+## AS Number
 
-## AS Number search
+An [AS number](https://www.cloudflare.com/learning/network-layer/what-is-an-autonomous-system/) is a group of IP addresses belonging and controlled by a single organization. The entire group of networks have a single unified routing policy. The [Internet Assigned Numbers Authority](https://www.iana.org/) (IANA) is the organization responsible for managing the assignment and distribution of AS numbers. The AS number's routing policies are used by [BGP](https://www.cloudflare.com/learning/security/glossary/what-is-bgp/) which is how Cloudflare's [anycast network](https://www.cloudflare.com/learning/cdn/glossary/anycast-network/) works. 
 
-### Overview
+By searching for an AS number, Cloudflare will return registration data such as its country, description and type. It will also display data such as domain count, top 10 domains and subnets.
```
Comment (deadlypants1973) [inline content/security-center/investigate/investigate-threats.md]: Same comment here as in domains. who is doing the searching? It sounds like Cloudflare is doing the searching

File: content/security-center/investigate/scan-limits.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
+---
+pcx_content_type: concept
+title: Scan limits
+meta:
+  description: Limits
+---
+
+# Scan limits
+
+URL scans are limited by search history, Public and Unlisted visibility and requests per second across different Cloudflare plans.
```
Comment (deadlypants1973) [inline content/security-center/investigate/scan-limits.md]:
```suggestion
URL scans are limited by search history, Public and Unlisted visibility, and requests per second across different Cloudflare plans. 
```

File: content/security-center/investigate/investigate-threats.md [hunk 10]
```diff
@@ -6,77 +6,56 @@ weight: 3
...
 
-## Analyze
+### Filters
 
-You can upload a JavaScript file to scan for malicious content using the **Analyze** feature.
-
-The scanner provides a percentage of integrity, a general measurement of safety, for the file. You can decide what percentage of integrity is safe to use.
-
-Contact your account team if you are interested in enabling this feature.
+While viewing the most recent scans, you have the ability to filter scan reports initiated from your Cloudflare account, `All account scans`, or `All global scans`, which includes all `Public` scans.
```
Comment (deadlypants1973) [inline content/security-center/investigate/investigate-threats.md]: The second part of this sentence seems disconnected from the first?

While viewing the most recent scans, you have the ability to filter scan reports initiated from your Cloudflare account. 

Is this listing the type of scans available?
`All account scans`, or `All global scans`, which includes all `Public` scans.

**Pull request merged**